source/basic.tex

```diff
@@ -3772,11 +3772,15 @@
 If any bit in the value representation has an indeterminate value,
 the object has an indeterminate value;
 otherwise, if any bit in the value representation has an erroneous value,
-the object has an erroneous value\iref{conv.lval}.
+the object has an erroneous value.
 \begin{note}
 Objects with static or thread storage duration are zero-initialized,
 see~\ref{basic.start.static}.
 \end{note}
+\begin{note}
+The behavior of lvalue-to-rvalue conversion\iref{conv.lval} is undefined
+if the erroneous value of an object is not valid for its type.
```

Comment:
I think I want the cross-reference at the end of that sentence, because conv.lval discusses not just lvalue-to-rvalue conversion, but also the "bad" situation the second part of the sentence talks about.